mlir/include/mlir/Dialect/Tosa/IR/TosaUtilOps.td

```diff
@@ -86,12 +86,13 @@ def Tosa_VariableOp : Tosa_Op<"variable", []> {
   let summary = "Defines a variable";
 
   let description = [{
-    Defines a new TOSA variable. This is a mutable value.
+    Defines a new TOSA variable.
+    This is a persistent mutable value across multiple TOSA graph invocations.
     Modifications are expressed using read/write semantics.
   }];
 
   let arguments = (ins
-    SymbolNameAttr:$name,
+    I32Attr:$uid,
```

Comment:
Although the spec specifies an uid the name was kind of useful to tell the truth.

Reply:
Yea. The other dialects are actually using the Name Strings. https://github.com/llvm/llvm-project/blob/f1598367b661e46c4ecc7dd8ea35f9eac79a654c/mlir/include/mlir/Dialect/MLProgram/IR/MLProgramOps.td#L134

Reply:
Updated to `SymbolNameAttr:$uid`